### PR TITLE
[DOCS-121] Data Union docs

### PR DESCRIPTION
**Pull request by juhah**
Implementing the Data Union docs.

- Intro to Data Unions
- Create a DU with Core
- Framework Roles
- Auth & Identity
- Create & Monitor
- Join & Withdraw
- UX Best Practices

https://share.goabstract.com/d27c137b-ec79-4223-97a2-1488c7cc582d

### CODE REVIEW

**Comment by juhah** [comment]
@mattatgit @fonty1 Please have a look, PR url is: http://streamr-marketplace-pr-24fa6910abaab3a53e133fe0ec152c9eb8bb67f0.s3-website-eu-west-1.amazonaws.com/docs/data-unions

**Comment by mattatgit** [comment]
@juhah Cheers, will do

**Comment by fonty1** [comment]
You have so many conflicts! 😄 

Hopefully you don't mind, I'll push a round of fixes for the recently merged docs restructure to this branch shortly while it's fresh in my head.

**Comment by Saorikaji2** [comment]
All looks good! There are few points need to fix. 

**Intro to Data Unions page** 
 • Removing the "Period" of "Add users and publish to   the Marketplace." it reflect all devices. 
 • Line rag see the image
Intro to Data Unions


**UX best Practices page (mobile)-Fiat currency display section**
 
• Font size "3225" = 26pt "DATA" = 13pt
• Grey back ground square hight will be 136pt 

UX Best Practices mobile

File: app/src/docs/components/Pages/DataUnions/UXBestPractices/index.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react'
+import DocsHelmet from '$docs/components/DocsHelmet'
+import DocsLayout from '$docs/components/DocsLayout'
+import UXBestPracticesContent from '$docs/content/dataUnions/uxBestPractices.mdx'
+
+const UXBestPractises = () => (
```
Comment (fonty1) [inline app/src/docs/components/Pages/DataUnions/UXBestPractices/index.jsx:8]:
```suggestion
const UXBestPractices = () => (
```

File: app/src/docs/components/Pages/DataUnions/UXBestPractices/index.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <DocsLayout>
+        <DocsHelmet pageTitle="Data Unions UX best practices" />
+        <section>
+            <UXBestPracticesContent />
+        </section>
+    </DocsLayout>
+)
+
+export default UXBestPractises
```
Comment (fonty1) [inline app/src/docs/components/Pages/DataUnions/UXBestPractices/index.jsx:17]:
```suggestion
export default UXBestPractices
```

**Comment by juhah** [comment]
@fonty1 Thanks for fixing the conflicts :) I renamed instances of `practise` as `practice`.

@Saorikaji2 Fixed the "Intro to Data Unions" page:
Screen Shot 2020-05-07 at 9 29 14

**Pull request merged**